tests/functional/conftest.py

```diff
@@ -418,6 +432,14 @@ def pyi_builder(tmpdir, monkeypatch, request, pyi_modgraph):
     # The value is same as the original value.
     monkeypatch.setattr('PyInstaller.config.CONF', {'pathex': []})
 
+    def del_temp_dir():
+        if request.node.rep_setup.passed:
+            if request.node.rep_call.passed:
+                if os.path.exists(tmp):
+                    shutil.rmtree(tmp)
+
+    if is_darwin:
```

Comment:
I'm just curious:
- Which errors to be avoided  on Windows?
- Why not on Linux?